bundle/DependencyInjection/Configuration.php

```diff
@@ -73,4 +76,106 @@ private function addAsynchronousIndexingSection(ArrayNodeDefinition $nodeDefinit
+                        ->normalizeKeys(false)
+                        ->validate()
+                            ->ifTrue($keyValidator)
+                            ->thenInvalid('Site root name must be of string type.')
+                        ->end()
+                        ->integerPrototype()
+                            ->beforeNormalization()->always(static fn ($v) => is_string($v) ? (int)$v : $v)->end()
+                            ->info('Content ID')
+                        ->end()
+                    ->end()
+                    ->arrayNode('languages_siteaccess_map')
+                        ->info('Language key mapped to page siteaccess')
+                        ->useAttributeAsKey('name')
+                        ->normalizeKeys(false)
+                        ->validate()
+                            ->ifTrue($keyValidator)
+                            ->thenInvalid('Page name must be of string type.')
+                        ->end()
+                        ->prototype('array')
+                            ->useAttributeAsKey('name')
+                            ->normalizeKeys(false)
+                            ->validate()
+                                ->ifTrue($keyValidator)
+                                ->thenInvalid('Language code must be of string type.')
+                            ->end()
+                            ->prototype('scalar')
+                                ->validate()
+                                    ->ifTrue(static fn ($v) => !is_string($v))
+                                    ->thenInvalid('Siteaccess name must be of string type.')
+                                ->end()
+                            ->end()
+                        ->end()
+                    ->end()
+                    ->scalarNode('host')
+                        ->info('Host to index page from, defined in .env files')
+                        ->validate()
+                            ->ifTrue(static fn ($v) => !is_string($v))
+                            ->thenInvalid('Host must be of string type.')
+                        ->end()
+                    ->end()
+                    ->arrayNode('config')
+                        ->info('Config for separating page text by importance of the html tags and classes, used to index content to separate solr fields')
+                        ->validate()
+                            ->ifTrue($keyValidator)
+                            ->thenInvalid('Array key (level of field importance) must be of string type.')
+                        ->end()
+                        ->prototype('array')
+                            ->useAttributeAsKey('name')
+                            ->normalizeKeys(false)
+                            ->scalarPrototype()
+                                ->validate()
+                                    ->ifTrue(static fn ($v) => !is_string($v))
+                                    ->thenInvalid('HTML tag and class must be of string type.')
```

Comment:
Let's call this "selector", like:

```
HTML selector must be of string type.
```

Reply:
resolved in https://github.com/netgen/ibexa-search-extra/commit/3a5d0eff4f07d403b79eee68b2e45a2a1a23781b

Reply:
resolved in 6f3976bb033b0d64e1a106e1abca274eed6ac40b